### PR TITLE
Jee integration and improvements

### PR DESCRIPTION
**Pull request by MaxShapiro32**
Updated application to work with both zOS mainframe and JEE backends, converted Mapbox Access Token to env variable, and changed project name to Example Health Analytics.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
 
-Summit's health records look very similar to the health records of most insurance companies.
-
-Here's a view a data analyst might see when they interact with the Summit Health Analytics Application:
+Here's a view a data analyst might see when they interact with the Example Health Analytics Application:
 
 ![](readme_images/screenshot.png)
 
-Summit has recently started understanding how data science/analytics on some of the patient records, might surface interesting insights. There is lots of talk about this among some of the big data companies.
+Example Health has recently started understanding how data science/analytics on some of the patient records, might surface interesting insights. There is lots of talk about this among some of the big data companies.
```
Comment (loafyloaf) [inline README.md]: unneeded comma?

File: README.md [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
 
-Summit has also heard a lot about cloud computing. There is a lot of legacy code in the mainframe, and it works well for now, but Summit thinks it may be a complimentary opportunity to explore some data science/analytics in the cloud.
+Example Health has also heard a lot about cloud computing. There is a lot of legacy code in the mainframe, and it works well for now, but they think it may be a complimentary opportunity to explore some data science/analytics in the cloud.
```
Comment (loafyloaf) [inline README.md]: maybe switch up one of the uses of 'a lot' in these sentences?

**Review by loafyloaf — APPROVED** [review]
just a few style notes, but nothing that blocks merging.  also, is it 'z/OS' or 'zOS'?

File: README.md (hunk 1 above)
Comment (MaxShapiro32) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (MaxShapiro32) [reply README.md]: Updated

**Comment by MaxShapiro32** [comment]
I think it is supposed to be 'z/OS' so I changed it in the README

**Pull request merged**